### ISSUE TITLE
De-dupe posts to slack

### ISSUE THREAD

**Issue by mikeyrcamp** [f3peakcity/f3_bot#16]
There is a case where user's get an error when they submit the backblast, but it is still added to the queue. We end up with AFAIK duplicate queue entries which shows up as duplicate posts (need to debug this a bit further). If the event queue sheets task could perform a match from the queue to ensure the message is not already in the database, and possibly check slack messages as well to see if the entry for that day has already been posted.

Need more details from Steaks to figure this one out. https://f3peakcity.slack.com/archives/C04F1C0HY2F/p1678708722427749

**Comment by ugotmarcus** [comment]
Correct.  Clicked Submit, received error, then clicked again and both posted.  I will know next time not to click, but it happens so fast and I just clicked quickly without much thought like most IT users do. :)
